src/Cell.css

```diff
@@ -53,7 +53,7 @@
   transform: rotate(270deg) scaleX(-1);
 } */
 
-.cell.flipped.rotated-0 {
+.cell.flipped {
```

Comment:
This commit does not belong to this pull request: it fixes different thing.

Reply:
Yeah. it's from my other pull request, just messed up the branches.